### ISSUE TITLE
Undefined constant "TL_MODE"

### ISSUE THREAD

**Issue by lizardat** [toteph42/spambot#2]
Contao 5.3
Manager hängt sich auf: 

Database Problems
Exception in file vendor/toteph42/spambot/src/Resources/contao/config/config.php on line 25 Undefined constant "TL_MODE"

Sofern ich recherchieren konnte, gibt es in Contao 5 kein TL_MODE mehr?

**Comment by toteph42** [comment]
Ja, stimmt. Plugin ist nicht 5.3 kompatibel. Ich habe es mir bereits angeschaut - ist ein größerer Aufwand, den ich nicht bereit bin in meiner Freizeit zu investieren.
